Application.mk

```diff
@@ -55,9 +55,12 @@ ifeq ($(BUILD_MODULE),y)
   # Revisit: This only works for gcc and clang.
   # Do other compilers have similar?
   COMPILER_RT_LIB = $(shell $(CC) $(ARCHCPUFLAGS) --print-libgcc-file-name)
-  ifneq ($(COMPILER_RT_LIB),)
-    LDLIBS += $(COMPILER_RT_LIB)
+  ifeq ($(wildcard $(COMPILER_RT_LIB)),)
```

Comment:
I'm still interested in what is an output of `$(shell $(CC) $(ARCHCPUFLAGS) --print-libgcc-file-name)` that hits `ifeq ($(wildcard $(COMPILER_RT_LIB)),)` to true.

Reply:
--print-libgcc-file-name: 
```
$ clang  --config armv8m.main_hard_fp_nosys --target=armv8m.main-none-eabi -mfloat-abi=hard  -march=armv8m.main+fp --print-libgcc-file-name

/home/archer/code/l61/ap/prebuilts/clang/linux/arm/lib/clang/13.0.0/lib/baremetal/libclang_rt.builtins-armv8m.main.a
```
library not found.
```
$ ls /home/archer/code/l61/ap/prebuilts/clang/linux/arm/lib/clang/13.0.0/lib/baremetal/libclang_rt.builtins-armv8m.main.a

ls: cannot access '/home/archer/code/l61/ap/prebuilts/clang/linux/arm/lib/clang/13.0.0/lib/baremetal/libclang_rt.builtins-armv8m.main.a': No such file or directory
```

--print-file-name:
```
$ clang  --config armv8m.main_hard_fp_nosys --target=armv8m.main-none-eabi -mfloat-abi=hard  -march=armv8m.main+fp --print-file-name libclang_rt.builtins-armv8m.main.a

/home/archer/code/l61/ap/prebuilts/clang/linux/arm/bin/../lib/clang-runtimes/armv8m.main_hard_fp/lib/libclang_rt.builtins-armv8m.main.a
```

library found.
```
$ ls /home/archer/code/l61/ap/prebuilts/clang/linux/arm/bin/../lib/clang-runtimes/armv8m.main_hard_fp/lib/libclang_rt.builtins-armv8m.main.a

/home/archer/code/l61/ap/prebuilts/clang/linux/arm/bin/../lib/clang-runtimes/armv8m.main_hard_fp/lib/libclang_rt.builtins-armv8m.main.a
```

Reply:
> what's the output of --print-libgcc-file-name and --print-file-name for the target?

@yamt